azure_jumpstart_ag/artifacts/AgLogonScript.ps1

```diff
@@ -288,7 +335,7 @@ Invoke-Command -VMName $VMnames -Credential $Credentials -ScriptBlock {
     $timeElapsed = 0
     do {
         Write-Host "INFO: Waiting for internet connection to be healthy on $hostname."
-        Start-Sleep -Seconds 5
+        sleep 5
```

Comment:
I think you missed the PR update where `sleep` was replaced with `Start-Sleep`

Reply:
I have figured out right after PR and updated already.